src/i-client-config.ts

```diff
@@ -1,4 +1,10 @@
-import { AttributeType, Flag, IAssignmentLogger, IAsyncStore } from '@eppo/js-client-sdk-common';
+import {
+  AttributeType,
+  ContextAttributes,
+  Flag,
+  IAssignmentLogger,
+  IAsyncStore,
+} from '@eppo/js-client-sdk-common';
```

Comment:
The lint is complaining about these imports being unused. Did you intent to re-export them?